android/PhysicalWeb/app/src/main/res/values/strings.xml

```diff
@@ -62,11 +54,6 @@
     <string name="title_activity_oob">OobActivity</string>
```

Comment:
Its not used in any java files, it is only used in the AndroidManifest. Can I just place the literal there?